Disable "Loading Renderer" message

DanielSant0s opened the pull request:
Disable message when loading a new renderer, less I/O, more performance :p
That message will appear when DEBUG mode is used.

DanielSant0s committed:
perf: Disable "Loading Renderer" message

Disable message when loading a new renderer, less I/O, more performance :p

rickgaiser commented:
The GL_FUNC_DEBUG function I added to make a trace of all OpenGL functions being called by the application, with all arguments being passed. This is not an OpenGL interface function.

The debug functions are defined here:
https://github.com/ps2dev/ps2stuff/blob/master/include/ps2s/debug_macros.h
They give more info like filename and line number and are enabled when compiling ps2gl or ps2stuff with `-DDEBUG`

Can you change from GL_FUNC_DEBUG to mDebugPrint ?

DanielSant0s committed:
Change GL_FUNC_DEBUG to mDebugPrint

DanielSant0s commented:
> The GL_FUNC_DEBUG function I added to make a trace of all OpenGL functions being called by the application, with all arguments being passed. This is not an OpenGL interface function.
> 
> The debug functions are defined here: https://github.com/ps2dev/ps2stuff/blob/master/include/ps2s/debug_macros.h They give more info like filename and line number and are enabled when compiling ps2gl or ps2stuff with `-DDEBUG`
> 
> Can you change from GL_FUNC_DEBUG to mDebugPrint ?


fair, changed
